ui/src/services/csrfTokenApi.ts

```diff
@@ -131,4 +137,21 @@ export function setupCsrfInterceptor(csrfTokenApi: CreateActions<CsrfTokenApi>,
 
         return config
     })
+
+    axiosInstance.interceptors.response.use(undefined, async (error: AxiosError) => {
```

Comment:
What does passing `undefined` here and a function as the second arg do? Is the first for handling success and the second for errors?

Reply:
Yep, exactly, the first handle success and second handles errors.